packages/gamut/src/BetaSticker/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme } from '@codecademy/gamut-styles';
+import React from 'react';
+
+import { Box, Text } from '..';
+
+type BetaStickerProps = {
+  className?: string;
+  children?: never;
+};
+
+export const BetaSticker: React.FC<BetaStickerProps> = (props) => {
+  return (
+    <Box
+      background="transparent"
+      boxShadow={`-2px 2px 0 ${theme.colors.navy}`}
+      display="inline-block"
+      height={26}
+      px={6 as any}
+      py={2 as any}
+    >
```

Comment:
If you'd like to avoid the casting - since these are fixed dimensions we can just specify a width - and center it with flex properties.

If that acts weirdly add `display="inline-block"` to the text :)

```suggestion
    <FlexBox
      inline
      center
      bg="transparent"
      boxShadow={({ colors }) => `-2px 2px 0 ${colors.secondary}`}
      height={26}
      width={52}
    >
```